test/Microsoft.Azure.WebJobs.Host.EndToEndTests/ServiceBusEndToEndTests.cs

```diff
@@ -60,106 +60,35 @@ public ServiceBusEndToEndTests()
             }
             finally
             {
-                Cleanup();
+                await Cleanup();
             }
         }
 
         [Fact]
-        public async Task ServiceBusEndToEnd_RestrictedAccess()
+        public async Task ServiceBusBinderTest()
         {
             try
             {
-                // Try running the tests using jobs that declare restricted access
-                // levels. We expect a failure.
-                FunctionListenerException expectedException = null;
-                try
-                {
-                    await ServiceBusEndToEndInternal(typeof(ServiceBusTestJobs_RestrictedAccess));
-                }
-                catch (FunctionListenerException e)
-                {
-                    expectedException = e;
-                }
-                Assert.NotNull(expectedException);
-
-                // Now create the service bus entities
-                string queueName = ResolveName(QueueNamePrefix + "1");
-                _namespaceManager.CreateQueue(queueName);
-
-                string topicName = ResolveName(TopicName);
-                _namespaceManager.CreateTopic(topicName);
+                var hostType = typeof(ServiceBusTestJobs);
+                var host = CreateHost(hostType);
+                var method = typeof(ServiceBusTestJobs).GetMethod("ServiceBusBinderTest");
 
-                string subscription1 = ResolveName(QueueNamePrefix + "topic-1");
-                _namespaceManager.CreateSubscription(topicName, subscription1);
-
-                string subscription2 = ResolveName(QueueNamePrefix + "topic-2");
-                _namespaceManager.CreateSubscription(topicName, subscription2);
-
-                // Test should now succeed
-                await ServiceBusEndToEndInternal(typeof(ServiceBusTestJobs_RestrictedAccess), verifyLogs: false);
-            }
-            finally
-            {
-                Cleanup();
-            }
-        }
+                int numMessages = 10;
+                var args = new { message = "Test Message", numMessages = numMessages };
+                await host.CallAsync(method, args);
+                await host.CallAsync(method, args);
+                await host.CallAsync(method, args);
 
-        [Fact]
-        public async Task ServiceBusEndToEnd_CreatesEntities()
```

Comment:
I see a few e2e tests were removed here -- why? Do they no longer apply?

Reply:
Dotnet service bus client library can't create new entities (queues, topics). That's why those tests were removed.